src/components/summary-list/template.njk

```diff
@@ -6,20 +6,27 @@
     {% endif -%}
   </a>
 {% endmacro -%}
+
+{# Determine if we need 2 or 3 columns #}
+{% set hasActions = false %}
+{% for row in params.rows %}
+  {% set hasActions = row.actions.items if row.actions.items else hasActions %}
```

Comment:
As this is a boolean, should we set it to `true if row.actions.items`, rather than the contents of `row.actions.items`?

Reply:
Thanks, sorted